clash_lib/src/proxy/tuic/handle_task.rs

```diff
@@ -141,7 +144,7 @@ impl TuicConnection {
         }
 
         match self.inner.heartbeat().await {
-            Ok(()) => tracing::trace!("[heartbeat]"),
+            Ok(()) => tracing::debug!("[heartbeat]"),
```

Comment:
maybe add more context to this line, such as [quic heartbeat] with other possible information (remote endpoint etc). it's hard to track where this is originated when logs are verbose